channeldb/db.go

```diff
@@ -9,16 +9,41 @@ import (
 	"sync"
 
 	"github.com/boltdb/bolt"
+	"github.com/go-errors/errors"
 	"github.com/roasbeef/btcd/btcec"
 	"github.com/roasbeef/btcd/chaincfg"
 	"github.com/roasbeef/btcd/wire"
 )
 
 const (
-	dbName = "channel.db"
+	dbName           = "channel.db"
+	dbFilePermission = 0600
+
+	// dbCorruptedPrefix is used for renaming the default name of the
+	// database if migration is failed.
+	dbCorruptedPrefix = "corrupted."
+
+	// dbBackupPrefix is used for constructing the name of the backup
+	// file of the database during migration process.
+	dbBackupPrefix = "backup."
 )
 
+// Migration is a function which changes the data accordingly to maintain the
```

Comment:
I think a comment long the lines of this may be a bit more accurate: 
> Migration is a function which takes a prior outdated version of the database instances and mutates the key/bucket structure to arrive at a more up-to-date version of the database.